src/test/HelmetTest.js

```diff
@@ -217,7 +217,20 @@ describe("Helmet", () => {
 
                 const htmlTag = document.getElementsByTagName("html")[0];
 
-                expect(htmlTag.attributes.length).to.equal(0);
+                expect(htmlTag.getAttribute("lang")).to.be.null;
+                expect(htmlTag.getAttribute("amp")).to.be.null;
```

Comment:
I think we still need the "And test that Helmet does not override attributes when argument not present in Helmet" case.